sklearn/ensemble/tests/test_forest.py

```diff
@@ -917,8 +917,6 @@ def check_class_weight_balanced_and_bootstrap_multi_output(name):
     # smoke test for subsample and balanced subsample
     clf = ForestClassifier(class_weight='balanced_subsample', random_state=0)
     clf.fit(X, _y)
-    clf = ForestClassifier(class_weight='subsample', random_state=0)
```

Comment:
I cannot see this: the tests need to be restored to check that the `class_weight='subsample'` option is still supported (although deprecated).